src/app/shared/service/http.service.ts

```diff
@@ -17,8 +17,8 @@ export class HttpService {
 		this.httpClient = httpClient;
 	}
 
-	public get(url): Observable<any> {
-		return this.httpClient.get(`${BASE_URL}${url}`);
+	public get(url, params?): Observable<any> {
+		return this.httpClient.get(`${BASE_URL}${url}`, {params});
```

Comment:
what is params in this case ? an object ? are you sending { params: { sort... } } ?
if so, send an object without the params attribute